### ISSUE TITLE
[BUG] Change the Release Channel according to the User's assigned release channel.

### ISSUE THREAD

**Issue by SecondNewtonLaw** [pizzaboxer/bloxstrap#1385]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Currently, as far as I'm concerned, Bloxstrap forces you into the LIVE branch, this is fine for most cases, except reciently, more and more A/B testing channels have been released, and I have found friends that even with brand new accounts get enrolled into release channels other than LIVE, even on brand new accounts. Considering what happened before with Roblox (Users reverting their assigned channels back to LIVE to cheat) I believe that this time around punishments may be delivered if a user is not assigned to their correct channel. It is just an assumption, but the feature request, to dumb it down to a sentence.

- [ ] Make it able for Bloxstrap to obtain the assigned channel to you as a user, and get the roblox version for that channel.

the new channels are along the lines of `zexpcontrol1`, `zexpcontrol2` and `zexpvariantpublic`, and I believe ignoring this may have a consequence, as available cheats right now (Such as Krampus) only hold support for the LIVE branch of Roblox, and reverting branches back to LIVE while being assigned to another one may be taken as a "Possible Cheater" flag for Roblox, although these are all asumptions, I believe it should be taken into account.

**Comment by cub-has-injected** [comment]
I used cheats before, and it would be useful when Roblox tries to force-update you (like example at early 2023 with Byfron update), but you can still can change it by yourself using this one thing:

- Go to the installation folder that Bloxstrap uses and change one line in Settings.json (from false to true):
```
  "OhHeyYouFoundMe": true
```
Now, you can go to https://clientsettings.roblox.com/v2/user-channel?binaryType=WindowsPlayer and check your current channel and change it in Bloxstrap at the Behavior section whenever you want.

**Comment by pizzaboxer** [comment]
> Currently, as far as I'm concerned, Bloxstrap forces you into the LIVE branch,

No it doesn't. By default, it follows whatever Roblox tries to set your channel to whenever you launch from the website.

> punishments may be delivered if a user is not assigned to their correct channel

No they absolutely will not. lol.
